mm2src/coins/z_coin.rs

```diff
@@ -1205,6 +1206,8 @@ impl MmCoin for ZCoin {
         Box::new(fut.boxed().compat())
     }
 
+    fn get_raw_transaction(&self, _req: RawTransactionRequest) -> RawTransactionFut { unimplemented!() }
```

Comment:
Zcash and its forks is UTXO coin. You can use `utxo_common::get_raw_transaction`.